terraform/tfschema/devicetrust/v1/device_terraform.go

```diff
@@ -174,11 +127,9 @@ func GenSchemaDeviceV1(ctx context.Context) (github_com_hashicorp_terraform_plug
 			Optional:    true,
 		},
 		"version": {
-			Computed:      true,
-			Description:   "Version is version",
-			Optional:      true,
-			PlanModifiers: []github_com_hashicorp_terraform_plugin_framework_tfsdk.AttributePlanModifier{github_com_hashicorp_terraform_plugin_framework_tfsdk.UseStateForUnknown()},
-			Type:          github_com_hashicorp_terraform_plugin_framework_types.StringType,
+			Description: "Version is the API version used to create the resource. It must be specified. Based on this version, Teleport will apply different defaults on resource creation or deletion. It ust be an integer prefixed by \"v\". For example: `v1`",
```

Comment:
There's a typo at
`It ust be an` -> `It must be an`

I sent a comment to the [docs PR](https://github.com/gravitational/teleport/pull/36774)